dev/run-tests.py

```diff
@@ -294,7 +294,8 @@ def build_spark_sbt(hadoop_version):
     sbt_goals = ["package",
                  "assembly/assembly",
                  "streaming-kafka-assembly/assembly",
-                 "streaming-flume-assembly/assembly"]
+                 "streaming-flume-assembly/assembly",
+                 "streaming-mqtt-assembly/assembly"]
```

Comment:
Ah, `test:package` doesn't contain the dependencies. Could you try this commit https://github.com/zsxwing/spark/commit/97244ec63937f6c23990f73b054af3d450344bb8 ? I have tested it locally.